docs/index.md

```diff
@@ -1,9 +1,23 @@
-# Welcome
+# Introduction to Percona Everest
 
-Percona Everest is an open source Database-as-a-Service (DBaaS) solution that automates day-one and day-two operations for Postgres, MySQL, and MongoDB databases within the Kubernetes clusters.
+PerconaEverest is an open-source private database-as-a-service that enables developers to quickly deploy code, scale deployments, and reduce database administration overhead. It also gives them control over their data, database configuration, and DBaaS costs.
```

Comment:
Percona Everest

Reply:
In Percona, we use "open source", non-hyphenated.

Reply:
Fixed